sdk/test/Services/DynamoDBv2/IntegrationTests/DataModelTests.cs

```diff
@@ -73,6 +73,92 @@ public void TestContextWithEmptyStringDisabled()
             TestEmptyStringsWithFeatureDisabled();
         }
 
+        /// <summary>
+        /// Tests that the DynamoDB operations can be invoked successfully based on the table info 
+        /// supplied only via class attributes.
+        /// </summary>
+        [TestMethod]
+        [TestCategory("DynamoDBv2")]
+        public void TestContext_DescribeTableDisabled_WithFullAttributes()
+        {
+            TableCache.Clear();
+            CleanupTables();
+            TableCache.Clear();
+
+            CreateContext(DynamoDBEntryConversion.V2, true, true);
+
+            TestHashRangeObjects<AnnotatedEmployee>();
```

Comment:
Are any other  "sub"-tests from `TestContextWithEmptyStringEnabled` relevant here?

I had only grabbed this one during the prototype since it looked like a good place to start, but I didn't think about the rest.

Reply:
I took a look at these tests and did not come across a use case that will potentially be affected via the addition of the `SkipDescribeTableLoad` flag.